logverification/verifyconsistency_test.go

```diff
@@ -33,98 +30,127 @@ import (
 //     and at the same positions, and you can trust this as the new head of the log.
 //  3. Get the hashes of peaks in your trusted log state and verify using a consistency proof based
 //     on a published signed state.
-func TestVerifyConsistency(t *testing.T) {
-	testContext, testGenerator, _ := integrationsupport.NewAzuriteTestContext(t, "TestVerifyConsistency")
-
-	// - Public key to verify the signature of the signed state (ephemeral keys are used for this demo)
-	// - Massif context that represents the backed-up merkle log data for a point in the past (containing 7 events)
-	// - Massif context that represents the head of the log (after 4 additional events appended)
-	//
-	// (!) A MassifContext provides access to the log entries in a single massif. A massif is a
-	// logical chunk of the merkle log, stored in an Azure blob.
-	publicVerificationKey, backupMassifContext, newMassifContext := integrationsupport.SetupTest(t, testContext, testGenerator)
-
-	hasher := sha256.New()
-	codec, err := massifs.NewRootSignerCodec()
-	require.NoError(t, err)
 
-	// First we verify the signature. In order to trust the data, we want to see that DataTrails has
-	// committed to the state of the merkle log by signing it.
-	logStateNow, signatureVerificationErr := verifySignature(
-		context.Background(), publicVerificationKey, hasher, codec, testContext.Storer,
-		&newMassifContext,
+// Then we verify the consistency of the data between our previously trusted log state
+// (accessible through backupMassifContext) and the current state (accessible through
+// newMassifContext.)
+
+// backupMassifContext points to the following merkle log with 7 leaves.
+//
+//	    6
+//	  /   \
+//	 2     5     9
+//	/ \   / \   / \
+// 0   1 3   4 7   8 10   <- Leaf Nodes
```

Comment:
```suggestion
//     0   1 3   4 7   8 10   <- Leaf Nodes
```

Reply:
I just mean the alignment is off in the ascii art I think

Reply:
double check for tabs and spaces, i also notice go fmt doesn't attempt to format `/** */` comments but will attempt to format `//` comments.